core/controlplane/config/templates/cluster.yaml

```diff
@@ -1261,6 +1261,14 @@ experimental:
   kube2IamSupport:
     enabled: false
 
+  # Use IPVS kube-proxy mode instead of [default] iptables one (requires Kubernetes 1.8.3+)
+  # This is intended to address performance issues of iptables mode for clusters with big number of nodes and services
+  ipvsProxy:
```

Comment:
Could you move this to `kubeProxy.ipvsMode`?
Sorry for taking your time but we're migrating away from the `experimental` settings because not only `experimental` but everything could change while we're in pre v1.0 😉